diff --git "a/C:\\Projekte\\stm32mp157a-avenger96.dts" "b/C:\\Projekte\\stm32mp157a-avenger96_sdmmc_v3v3.dts"
index 7b8c3f2..b1a1b6c 100644
--- "a/C:\\Projekte\\stm32mp157a-avenger96.dts"
+++ "b/C:\\Projekte\\stm32mp157a-avenger96_sdmmc_v3v3.dts"
@@ -223,8 +223,8 @@
 
 			vdd_sd: ldo5 {
 				regulator-name = "vdd_sd";
-				regulator-min-microvolt = <2900000>;
-				regulator-max-microvolt = <2900000>;
+				regulator-min-microvolt = <3300000>;
+				regulator-max-microvolt = <3300000>;
 				interrupts = <IT_CURLIM_LDO5 0>;
 				interrupt-parent = <&pmic>;
 				regulator-boot-on;
